describe.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+
+import { describe as describeSclElement } from "./describe.js";
+
+const testScl = new DOMParser().parseFromString(
+  `<SCL
+      xmlns="http://www.iec.ch/61850/2003/SCL"
+      xmlns:sxy="http://www.iec.ch/61850/2003/SCLcoordinates"
+      xmlns:ens="http://somevalidURI"
+    >
+      <Private type="someType" desc="someDesc" sxy:x="10" ens:some="someOtherNameSpace">
+        <![CDATA[some comment]]>
+        <IED name="somePrivateIED"/>
+      </Private>
+    </SCL>`,
+  "application/xml"
+);
+
+const privateElement = testScl.querySelector("Private")!;
+const sclElement = testScl.querySelector("SCL")!;
+
+describe("Describe SCL elements function", () => {
+  it("returns undefined with missing describe function", () =>
+    expect(describeSclElement(sclElement)).to.be.undefined);
```

Comment:
Using a non-SCL element here (like `testSCL.createElement('InvalidTagName')` or something) would be more stable, meaning the test won't have to be changed once the `describe` function is ready.

Reply:
I would like to leave it in to cover the used case that will be there for a while I guess. And having this one test erased would be ok for me. Your comment here reminds me though to add a test for non SCL elements.